fava_investor/modules/performance/accounts_config.py

```diff
@@ -0,0 +1,41 @@
+import re
+from typing import List
+
+from core import FavaLedger
+
+from fava_investor.modules.performance.balances import ConfigDict
+
+
+class AccountsConfig:
+    def __init__(self, value: List[str], internal: List[str], external: List[str]):
+        """
+        Full lists of all accounts considered value/internal/external. useful when building queries
+        for matching account or these with joinstr(other_accounts)
+        """
+        self.value = value
+        self.internal = internal
+        self.external = external
+
+    @classmethod
+    def from_dict(cls, ledger: FavaLedger, config: ConfigDict) -> 'AccountsConfig':
```

Comment:
Let's move this into balances.py as a separate function. It could look like:
def filter_accounts(accounts, value_regexp, int_regexp, ext_regexp)
  ....
  return(value_accounts, internal_accounts, external_accounts)